llvm/test/CodeGen/AMDGPU/dagcombine-fmul-sel.ll

```diff
@@ -0,0 +1,2616 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+;RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx700 < %s | FileCheck -check-prefix=GFX7 %s
```

Comment:
Should rebase on top of the test base. Patch lgtm with nit but I can't easily check the effect in the output here